### ISSUE TITLE
Pytorch中的Conv1d和Conv3d | 小王同学

### ISSUE THREAD

**Issue by a171232886** [a171232886/a171232886.github.io#23]
https://a171232886.github.io/2021/11/22/Pytorch%E4%B8%AD%E7%9A%84Conv1d%E5%92%8CConv3d/ 

 Conv1d
 torch.nn.Conv1d(in_channels, out_channels, kernel_size, stride=1, padding=0, dilation=1, groups=1, bias=True, padding_mode='zeros')
 和Conv2d处理[N,C,H,W]相比，只是处理处理的数据为[N,C,L]，其余相同。
